src/UnitySampleProject/Assets/Scripts/Prefabs/Web3AuthWallet/Interfaces/IEthereumService.cs

```diff
@@ -0,0 +1,14 @@
+using System.Threading.Tasks;
+using Nethereum.RPC.Eth.DTOs;
+
+namespace Prefabs.Web3AuthWallet.Interfaces
+{
+    public interface IEthereumService
+    {
+        string GetAddressW3A(string privateKey);
+
+        Task<string> CreateAndSignTransactionAsync(TransactionInput txInput);
```

Comment:
Just a suggestion, we might want to follow standard security practises of not allowing the signed transaction to exist without having sent it immedietly after signing